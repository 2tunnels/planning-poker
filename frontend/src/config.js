export const WS_BASE_URL = process.env.REACT_APP_WS_BASE;
export const API_BASE_URL = process.env.REACT_APP_API_BASE;
export const VOTE_OPTIONS = ["0", "1/2", "1", "2", "3", "5", "8", "13"];
