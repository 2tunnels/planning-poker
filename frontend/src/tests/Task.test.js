import { render } from "./test-utils";

import TaskComponent from "../components/Task";
import { testGameId } from "./config";

const task = {
  id: 1,
  content: "Task 1, as a user",
  complexity: 0,
};
const gameId = testGameId;
const voteDistribution = {
  1: {
    0: 1,
    0.5: 0,
    1: 0,
  },
};

describe("Task component", () => {
  it("renders to snapshot", () => {
    const { container, getByText } = render(
      <TaskComponent
        task={task}
        gameId={gameId}
        votesDistribution={voteDistribution}
      />
    );
    expect(container).toMatchSnapshot();
    expect(getByText("Task 1, as a user")).toBeInTheDocument();
  });
});
