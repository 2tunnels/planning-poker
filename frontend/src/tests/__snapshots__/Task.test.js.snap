// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Task component renders to snapshot 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-4"
  >
    <div
      class="MuiBox-root MuiBox-root-5"
      xs="12"
    >
      <h4
        class="MuiTypography-root MuiTypography-h4"
      >
        Task 1, as a user
      </h4>
    </div>
    <div
      class="MuiBox-root MuiBox-root-6 makeStyles-cards-3"
    >
      <div
        class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          class="MuiCardHeader-root makeStyles-cardHeader-2"
        >
          <div
            class="MuiCardHeader-content"
          >
            <span
              class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            />
          </div>
        </div>
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardContent-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              0
            </h6>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          class="MuiCardHeader-root makeStyles-cardHeader-2"
        >
          <div
            class="MuiCardHeader-content"
          >
            <span
              class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            />
          </div>
        </div>
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardContent-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              0.5
            </h6>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          class="MuiCardHeader-root makeStyles-cardHeader-2"
        >
          <div
            class="MuiCardHeader-content"
          >
            <span
              class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            />
          </div>
        </div>
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardContent-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              1
            </h6>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          class="MuiCardHeader-root makeStyles-cardHeader-2"
        >
          <div
            class="MuiCardHeader-content"
          >
            <span
              class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            />
          </div>
        </div>
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardContent-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              2
            </h6>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          class="MuiCardHeader-root makeStyles-cardHeader-2"
        >
          <div
            class="MuiCardHeader-content"
          >
            <span
              class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            />
          </div>
        </div>
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardContent-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              3
            </h6>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          class="MuiCardHeader-root makeStyles-cardHeader-2"
        >
          <div
            class="MuiCardHeader-content"
          >
            <span
              class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            />
          </div>
        </div>
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardContent-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              5
            </h6>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          class="MuiCardHeader-root makeStyles-cardHeader-2"
        >
          <div
            class="MuiCardHeader-content"
          >
            <span
              class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            />
          </div>
        </div>
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardContent-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              8
            </h6>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          class="MuiCardHeader-root makeStyles-cardHeader-2"
        >
          <div
            class="MuiCardHeader-content"
          >
            <span
              class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            />
          </div>
        </div>
        <button
          class="MuiButtonBase-root MuiCardActionArea-root"
          tabindex="0"
          type="button"
        >
          <div
            class="MuiCardContent-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              13
            </h6>
          </div>
          <span
            class="MuiCardActionArea-focusHighlight"
          />
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</div>
`;
